nilearn/input_data/fmriprep_confounds_strategy.py

```diff
@@ -145,22 +145,22 @@ def fmriprep_confounds_strategy(img_files, denoise_strategy="simple",
        strategies. Parameters with `*` denote customisable parameters. Please
        see :func:`nilearn.input_data.fmriprep_confounds`.
 
-        ========= ========= ====== ====== ============= ===== ========= \
-        ================ ============== ========= ========= ======
-        strategy  high_pass motion wm_csf global_signal scrub fd_thresh \
-        std_dvars_thresh compcor        n_compcor ica_aroma demean
-        ========= ========= ====== ====== ============= ===== ========= \
-        ================ ============== ========= ========= ======
-        simple    True      full*  basic* None*         N/A   N/A       \
-        N/A              N/A            N/A       N/A       True*
-        scrubbing True      full*  full   None*         5*    0.2*      \
-        3*               N/A            N/A       N/A       True*
-        compcor   True      full*  N/A    N/A           N/A   N/A       \
-        N/A              anat_combined* all*      N/A       True*
-        ica_aroma True      N/A    basic* None*         N/A   N/A       \
-        N/A              N/A            N/A       full      True*
-        ========= ========= ====== ====== ============= ===== ========= \
-        ================ ============== ========= ========= ======
+        ========= ========= ====== ====== ============= ===== ============ \
```

Comment:
It looks like the rendering of the table is not working anymore.

Reply:
Now it's fixed!